pkg/plugin/v3/webhook.go

```diff
@@ -96,6 +100,15 @@ func (p *createWebhookPlugin) Validate() error {
 			" --programmatic-validation and --conversion to be true", p.commandName)
 	}
 
+	if p.conversion {
+		if p.spoke == "" || p.hub == "" {
+			return fmt.Errorf("Hub and Spoke version need to be specified when using conversion webhooks")
+		}
+		if p.spoke == p.hub {
+			return fmt.Errorf("Cannot have the same hub and spoke version")
+		}
+	}
+
```

Comment:
Why should we make mandatory inform them?

Reply:
Because it depends on the user on what version they want to have as the hub. Without it, I think it wouldn't be fair to randomly chose a hub version and a non-storage version.

Reply:
Can we keep the current behaviour when these values are not informed?